retrieval_service/datastore/providers/postgres_test.py

```diff
@@ -12,144 +12,431 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import asyncio
 from collections import OrderedDict
+from datetime import datetime
+from ipaddress import IPv4Address
 from typing import Dict, cast
 
-import asyncpg
 import pytest
+import pytest_asyncio
+from langchain.embeddings import VertexAIEmbeddings
 
 import models
 
+from .. import datastore
 from . import postgres
+from .helpers import get_env_var
 
+DB_USER = get_env_var("DB_USER", "name of a postgres user")
+DB_PASS = get_env_var("DB_PASS", "password for the postgres user")
+DB_NAME = get_env_var("DB_NAME", "name of a postgres database")
+DB_HOST = get_env_var("DB_HOST", "ip address of a postgres database")
```

Comment:
Created inidividual fixtures on them. Will move to a conftest.py file in the future when more provider tests is added so that these fixtures can be shared across different providers.